### PR TITLE
Update JupyterLab version

### PR DESCRIPTION
**Pull request by VaishnaviHire**
Jira Issue: https://issues.redhat.com/browse/RHODS-2162

### CODE REVIEW

**Comment by VaishnaviHire** [comment]
Holding this until all the other PRs with this change get merged

/hold

**Comment by VaishnaviHire** [comment]
cc: @crobby

**Comment by jgarciao** [comment]
We want to support RHODS 1.2.0 and 1.3.0 in our automation framework. In order to do it,  soon PR #125 will be merged and you'll be able to use the new keyword  `RHODS Version Is Greater Or Equal Than`

Example:
```
${version-check} =  RHODS Version Is Greater Or Equal Than  1.3.0
IF  ${version-check}==True
  Run Keyword And Continue On Failure  Run Cell And Check Output  !pip show jupyterlab | grep Version:  Version: 3.2.4
ELSE
  Run Keyword And Continue On Failure  Run Cell And Check Output  !pip show jupyterlab | grep Version:  Version: 3.0.16
END
```

File: tests/Tests/500__jupyterhub/minimal-pytorch-test.robot
Comment (jgarciao) [inline tests/Tests/500__jupyterhub/minimal-pytorch-test.robot:47]: See above comment about how to handle different versions

File: tests/Tests/500__jupyterhub/minimal-tensorflow-test.robot
Comment (jgarciao) [inline tests/Tests/500__jupyterhub/minimal-tensorflow-test.robot:39]: See above comment about how to handle different versions

**Comment by jgarciao** [comment]
The keyword was finally added in #137  and it was called `Is RHODS Version Greater Or Equal Than
`
Example:
```
${version-check} =  Is RHODS Version Greater Or Equal Than  1.3.0
IF  ${version-check}==True
  Run Keyword And Continue On Failure  Run Cell And Check Output  !pip show jupyterlab | grep Version:  Version: 3.2.4
ELSE
  Run Keyword And Continue On Failure  Run Cell And Check Output  !pip show jupyterlab | grep Version:  Version: 3.0.16
END
```

**Comment by VaishnaviHire** [comment]
> The keyword was finally added in #137 and it was called `Is RHODS Version Greater Or Equal Than ` Example:
> 
> ```
> ${version-check} =  Is RHODS Version Greater Or Equal Than  1.3.0
> IF  ${version-check}==True
>   Run Keyword And Continue On Failure  Run Cell And Check Output  !pip show jupyterlab | grep Version:  Version: 3.2.4
> ELSE
>   Run Keyword And Continue On Failure  Run Cell And Check Output  !pip show jupyterlab | grep Version:  Version: 3.0.16
> END
> ```

I have updated the PR with the version keyword

**Review by jgarciao — APPROVED** [review]
The changes look good to me. I think you need to rebase from master and test this PR with jenkins.

**Comment by VaishnaviHire** [comment]
The changes were made in tests with ids ODS-207 and ODS-218, both of which passed in Jenkins
https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/274/TestReport/

**Review by lugi0 — APPROVED** [review]
LGTM and worked well on 1.3, needs final test on 1.4

**Review by lugi0 — APPROVED** [review]
LGTM

**Comment by lugi0** [comment]
@VaishnaviHire I hope you don't mind but I've pushed a commit on your branch to also include an updated version check for tensorflow and tensorboard in the tf image

**Review by lugi0 — APPROVED** [review]

**Review by vasukulkarni — APPROVED** [review]

**Pull request merged**
